frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -146,25 +151,47 @@ const Status = ({
 
 type TAmountsProps = {
   amount: IAmount;
+  fee: IAmount;
   type: TTransactionType;
 }
 
 const Amounts = ({
   amount,
+  fee,
   type,
 }: TAmountsProps) => {
   const { defaultCurrency } = useContext(RatesContext);
   const conversion = amount?.conversions && amount?.conversions[defaultCurrency];
   const sign = getTxSign(type);
   const txTypeClass = `txAmount-${type}`;
   const conversionPrefix = amount.estimated ? '\u2248' : null; // ≈
+
+  // const [fee, setFee] = useState<IAmount>();
+
+  // useEffect(() => {
+  //   if (
+  //     type === 'send_to_self'
+  //     && !fee?.amount
+  //     && accountCode && internalID
+  //   ) {
+  //     getTransaction(accountCode, internalID).then(transaction => {
+  //       setFee(transaction?.fee);
+  //     });
+  //   }
+  // }, [accountCode, fee, internalID, type]);
```

Comment:
If we don't have the fee in the /transactions endpoint we would need to fetch it somehow, with this PR we need it for send to self tx.